jobfile/job_file.go

```diff
@@ -13,6 +11,9 @@ import (
 	"strconv"
 	"strings"
 	"syscall"
+
+	"github.com/dshearer/jobber/common"
+	yaml "gopkg.in/yaml.v1"
```

Comment:
I think you'll need to use the Godeps path here ("github.com/dshearer/jobber/Godeps/_workspace/src/gopkg.in/yaml.v2")